llpc/include/llpc.h

```diff
@@ -109,14 +109,26 @@ struct ShaderModuleBuildOut {
   ShaderModuleData *pModuleData; ///< Output shader module data (opaque)
 };
 
+enum StageCacheInfo : uint8_t {
+  Invalid = 0, ///< Stage cache miss/hit information is invalid.
+  CacheMiss,   ///< Stage cache miss.
+  CacheHit,    ///< Stage cache hit.
+};
+
 /// Represents output of building a graphics pipeline.
 struct GraphicsPipelineBuildOut {
   BinaryData pipelineBin; ///< Output pipeline binary data
+  // Google - To support http://go/teamgob/519403 and http://b/138657311.
+  bool pipelineCacheHit;                               ///< Output if pipeline cache hits
+  StageCacheInfo stageCacheAccesses[ShaderStageCount]; ///< Output if stage cache hits
```

Comment:
I can't parse these two comments. Maybe something like `Indicates whether pipeline cache access was a hit`?

Reply:
I agree.  You should output a more descriptive comment like:

```
bool pipelineCacheHit;                               // True if the pipeline was found in the cache.
StageCacheInfo stageCacheAccesses[ShaderStageCount]; // True if an unlinked shader was found in the cache for the stage.
```

Reply:
done